### DISCUSSION TITLE
Mendeley Support docx+citations

### DISCUSSION THREAD

**Discussion by thomjur** [jgm/pandoc#10286] [Q&A]
Hi,

I recently came across my first docx file where someone used Mendeley to create the citations. According to the pandoc website, docx+citations should support citations created with Mendeley's Word-plugin:

> When citations is enabled in docx, citations inserted by Zotero or Mendeley or EndNote plugins will be parsed as native pandoc citations. (Otherwise, the formatted citations generated by the bibliographic software will be parsed as regular text.)

However, it does not work. I checked the docx-xml, and indeed different from Zotero-files, there are no embedded bibliographical information:

This is Mendeley Word-XML: 

```xml
<w:sdt>
<w:sdtPr>
<w:rPr>
<w:rFonts w:ascii="Times New Roman" w:hAnsi="Times New Roman" w:cs="Times New Roman"/>
<w:color w:val="000000"/>
<w:sz w:val="24"/>
<w:szCs w:val="24"/>
</w:rPr>
<w:tag w:val="MENDELEY_CITATION_v3_"/>
<w:id w:val="-303390472"/>
<w:placeholder>
<w:docPart w:val="DefaultPlaceholder_-1854013440"/>
</w:placeholder>
</w:sdtPr>
<w:sdtContent>
<w:r w:rsidR="00346226" w:rsidRPr="00346226">
<w:rPr>
<w:rFonts w:ascii="Times New Roman" w:hAnsi="Times New Roman" w:cs="Times New Roman"/>
<w:color w:val="000000"/>
<w:sz w:val="24"/>
<w:szCs w:val="24"/>
</w:rPr>
<w:t>(Dittrich, 1998)</w:t>
</w:r>
</w:sdtContent>
</w:sdt>

``` 

And this when using Zotero-plugin:

```xml
<w:r w:rsidR="00275102">
<w:fldChar w:fldCharType="begin"/>
</w:r>
<w:r w:rsidR="00275102">
<w:instrText xml:space="preserve"> ADDIN ZOTERO_ITEM CSL_CITATION {"citationID":"z2XM0m9E","properties":{"formattedCitation":"(Jurczyk 2019)","plainCitation":"(Jurczyk 2019)","noteIndex":0},"citationItems":[{"id":18541,"uris":["http://zotero.org/users/5235308/items/NMAS7RQ8"],"itemData":{"id":18541,"type":"chapter","container-title":"Bild Macht Religion","event-place":"Bochum","language":"ger","note":"Citation Key: jurczyk_christliche_2019","page":"114–118","publisher":"Kunstmuseum Bochum","publisher-place":"Bochum","title":"Christliche Bilderkontroversen vor dem byzantinischen Bilderstreit","author":[{"family":"Jurczyk","given":"Thomas"}],"editor":[{"family":"Golinski","given":"Hans Günter"},{"family":"Radermacher","given":"Martin"}],"issued":{"date-parts":[["2019"]]},"citation-key":"jurczyk_christliche_2019"}}],"schema":"https://github.com/citation-style-language/schema/raw/master/csl-citation.json"} </w:instrText>
</w:r>
<w:r w:rsidR="00275102">
<w:fldChar w:fldCharType="separate"/>
</w:r>
<w:r w:rsidR="00275102" w:rsidRPr="00275102">
<w:rPr>
<w:rFonts w:ascii="Aptos" w:hAnsi="Aptos"/>
</w:rPr>
<w:t>(Jurczyk 2019)</w:t>
</w:r>
<w:r w:rsidR="00275102">
<w:fldChar w:fldCharType="end"/>
</w:r>

```

My question is: Am I missing something here? Can the Mendeley-plugin embedd citations just like Zotero-plugin does?

Thank you all for your help!

**Comment by jgm** [comment]
See https://github.com/jgm/pandoc/issues/7886 for the issue adding Mendeley.
Does your document contain something like what is described there?
We are looking for the part that contains a JSON CSL bibliography.

**Reply by jgm** [reply]
There should be something in there marked `CSL_CITATION`.

It's possible that Mendeley has changed how they do this since that commit.

**Reply by thomjur** [reply]
Thank you for your reply! I'm new to Mendeley (only created an acc for this issue), but at least in the documents I'm creating, I can't find any embedded JSON-CSL; it looks completely different compared to #7886  (see my initial post). I also couldn't find any additional options like "Embed citations as JSON CSL" in the plugin. However, I might be overlooking something. Perhaps someone here is working with Mendeley, otherwise I may need to dive into the Mendeley plugin's release history... 😢
